### ISSUE TITLE
Add flag that allows suppresing report paths output if project has no errors

### ISSUE THREAD

**Issue by Naitbit** [autonomousapps/dependency-analysis-android-gradle-plugin#146]
As a dev that has project with almost 100 subprojects I get a lot of
```
> Task :analytics:adviceDebug
See machine-readable report at /home/user/projects/hello_world_project/analytics/build/reports/dependency-analysis/debug/advice.json
See pretty report at /home/user/projects/hello_world_project/analytics/build/reports/dependency-analysis/debug/advice-pretty.json
See machine-readable console report at /home/user/projects/hello_world_project/analytics/build/reports/dependency-analysis/debug/advice-console.json
See pretty console report at /home/user/projects/hello_world_project/analytics/build/reports/dependency-analysis/debug/advice-console-pretty.json

> Task :logger:adviceDebug
See machine-readable report at /home/user/projects/hello_world_project/logger/build/reports/dependency-analysis/debug/advice.json
See pretty report at /home/user/projects/hello_world_project/logger/build/reports/dependency-analysis/debug/advice-pretty.json
See machine-readable console report at /home/user/projects/hello_world_project/logger/build/reports/dependency-analysis/debug/advice-console.json
See pretty console report at /home/user/projects/hello_world_project/logger/build/reports/dependency-analysis/debug/advice-console-pretty.json
```
lines.
Since there is nothing to change in those project, reports are not useful and I have to skim/search for issues. 

**Describe the solution you'd like**
Provide a flag/setting that would allow either completely skipping printing report paths to stdout, or skipping them if plugin has no advice for given project.

**Comment by stephanenicolas** [comment]
We could even completely remove the output. There is no need for a flag.

**Comment by autonomousapps** [comment]
I will simply suppress the output if there is no advice for a given project. Expect this to land on master soon and be released sometime this week.

**Comment by autonomousapps** [comment]
Fixed by 2fe922de1939ed1be586b655df9fa558be7af1b3
